changelogs/fragments/6476-new-keycloak-user.module.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
+  - The alias username for the auth_username parameter of keycloak module_utils have been removed because it conflicted with the username parameter of the new keycloak user module (https://github.com/ansible-collections/community.general/pull/6476).
```

Comment:
Please note that breaking changes are not accepted with a long deprecation period.

Instead of removing that alias for all modules, you should just remove that alias for the new module. In `DOCUMENTATION` you can specify
```
options:
    auth_username:
        aliases: []
```
to override it, and you will have to modify the return value of `keycloak_argument_spec()` in the module to remove that alias as well.